user-guide/Reference/faq/IDs_of_views_elements_services_redundancy_groups_and_parameters.md

```diff
@@ -25,11 +25,11 @@ To find this ID in DataMiner Cube:
 
 1. Open the view containing the element, service or redundancy group.
 
-1. On the view card, go to the data side.
+1. Under BELOW THIS VIEW there is a page for each type of item that can be contained within the view. Using the ALL page will display all items within the view.
```

Comment:
If you add something as a step in a procedure, it must be an action, not a description of the UI. That means that this needs to be rephrased as something you want the user to do: 
1. In the navigation panel on the left, go to *BELOW THIS VIEW* > *All*.